src/Console/Exception/ConsoleException.php

```diff
@@ -29,5 +29,5 @@ class ConsoleException extends CakeException
      *
      * @var int
```

Comment:
These annotations are quite redundant now. We could remove them after updating our CS and API docs builder.

Reply:
I disabled the CS sniff for "useless annotation" to avoid annoying merge errors until we're done making api/docs changes to 4.3.